### ISSUE TITLE
Getting build errors 

### ISSUE THREAD

**Issue by AlmogRnD** [ivpusic/react-native-image-crop-picker#633]
Getting nothing but issues, I followed the installation guide but I keep getting different
<img width="302" alt="screen shot 2018-03-05 at 12 19 46 pm" src="https://user-images.githubusercontent.com/169507/36956001-db3c417c-207f-11e8-88bc-a0ee079c4fcd.png">
 errors

**Comment by ivpusic** [comment]
I just released a new version with updated installation instructions. If you follow step by step instructions from there, it will compile and run properly. https://github.com/ivpusic/react-native-image-crop-picker/releases/tag/v0.20.0

**Comment by AlmogRnD** [comment]
I did this but now I'm getting Lexical or Preprocessor issues 
`#import <React/RCTDefines.h> 'React/RCTDefines.h' file not found`

Happing at #import "ImageCropPicker.h" will open a new issue 

<img width="259" alt="screen shot 2018-03-06 at 11 52 22 am" src="https://user-images.githubusercontent.com/169507/37008140-da589372-2134-11e8-8f22-e02fa86452d5.png">

https://github.com/ivpusic/react-native-image-crop-picker/issues/635

**Comment by ivpusic** [comment]
can you show me you Pods file?

**Comment by AlmogRnD** [comment]
This is my entire pod file 

```
# Uncomment the next line to define a global platform for your project
 platform :ios, '8.0'

target 'ConstructionCloud' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for ConstructionCloud
  rn_path = '../node_modules/react-native'

    pod 'yoga', path: "#{rn_path}/ReactCommon/yoga/yoga.podspec"
    pod 'React', path: rn_path, subspecs: [
      'Core',
      'RCTActionSheet',
      'RCTAnimation',
      'RCTGeolocation',
      'RCTImage',
      'RCTLinkingIOS',
      'RCTNetwork',
      'RCTSettings',
      'RCTText',
      'RCTVibration',
      'RCTWebSocket'
    ]

    pod 'RNImageCropPicker', :path =>  '../node_modules/react-native-image-crop-picker'
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    if target.name == "React"
      target.remove_from_project
    end
  end
end
```

**Comment by ivpusic** [comment]
the only difference is that I don't have `use_frameworks!` in my Pods file. I followed instructions from readme step-by-step and it compiled and runs without problems.

**Comment by AlmogRnD** [comment]
Let me try without frameworks

**Comment by AlmogRnD** [comment]
Now I'm getting 

ld: 24 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by AlmogRnD** [comment]
This is the full error 

```
duplicate symbol _OBJC_IVAR_$_ImageResult._data in:
    /Users/Almog/Library/Developer/Xcode/DerivedData/ConstructionCloud-bcoecdtxeoikdpcgmgahscvuemda/Build/Products/Debug-iphonesimulator/RNImageCropPicker/libRNImageCropPicker.a(ImageCropPicker.o)
    /Users/Almog/Library/Developer/Xcode/DerivedData/ConstructionCloud-bcoecdtxeoikdpcgmgahscvuemda/Build/Products/Debug-iphonesimulator/libimageCropPicker.a(ImageCropPicker.o)
ld: 24 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by ivpusic** [comment]
maybe you linked it. Linking is only necessary if not using cocoapods. Try to unlink it.

**Comment by AlmogRnD** [comment]
Well I did run react-native link which I would have to do at some point for other packages so it shouldn't link this package. I did try to unlink it and I get the following error 


~/Full-Time-Employment/Construction-Cloud/RnD/CCMobileApp(branch:w3.2.7*) » react-native unlink react-native-image-crop-picker                    Almog@Almogs-MacBook-Pro
Scanning folders for symlinks in /Users/Almog/Full-Time-Employment/Construction-Cloud/RnD/CCMobileApp/node_modules (8ms)
rnpm-install info Unlinking react-native-image-crop-picker ios dependency 
rnpm-install ERR! It seems something went wrong while unlinking. Error: Cannot read property 'podfile' of undefined 

Cannot read property 'podfile' of undefined

**Comment by ivpusic** [comment]
`react-native link` without providing package name is not a good idea these days because many of third party modules are not playing well with `react-native link`, so I suggest to always link specific package. `Cannot read property 'podfile' of undefined` is something I see for the first time :/. Would be great if you can do some investigation what could be wrong here.

**Comment by AlmogRnD** [comment]
I'm not really seeing anything else that would hit at whats wrong, are you testing on the latest react version v0.54.0?

**Comment by AlmogRnD** [comment]
@ivpusic was to fix the Cannot read property 'podfile' of undefined I deleted the pod file then run the unlink command on the package added back the pod file and did pod install

**Comment by AlmogRnD** [comment]
But still getting 
ld: 24 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation) 

so it was not the linking

**Comment by ivpusic** [comment]
yes, I am using rn 0.54.0. Maybe open the project in Xcode, and check if react-native-image-crop-picker is still in Libraries folder. If yes, remove it. Also, check if libRNImageCropPicker is in Build Phases > Link binary with images, if yes, remove it.

**Comment by AlmogRnD** [comment]
Thanks working now

**Comment by hoscarcito** [comment]
Sorry to reopen but if the "link" is necessary for the Android build, what should we do?

**Comment by sahil290791** [comment]
Just remove `pod 'RNImageCropPicker', :path =>  '../node_modules/react-native-image-crop-picker'` from **Podfile** if you are going to use link command.

**Comment by tiaaaa123** [comment]
What if I need to use `use_frameworks!` Because of another pods?

**Comment by r01010010** [comment]
This is a nightmare, I'm starting to think it was a really bad idea to trust in react-native.

**Comment by musicode** [comment]
how to resolve it, I can't add any framework.

**Comment by musicode** [comment]
@ivpusic 

can you help me? how to resolve `#import <React/RCTDefines.h> 'React/RCTDefines.h' file not found`, even though I only added one line:

```
use_frameworks!
```

**Comment by letssave** [comment]
Hey @ivpusic,

Any news on the issue ? Did you resolved it ?

Thanks!

**Comment by edreyyo** [comment]
+1 Any updates about using this package with `use_frameworks!`, @ivpusic? This is basically the only cross-platform image picker package since `react-native-image-picker` is no longer maintained.

**Comment by hahtml** [comment]
Is it possible to install the package manually for ios?

**Comment by edreyyo** [comment]
@hahtml Apparently so, but the instructions are unclear (["Drag and drop the ios/ImageCropPickerSDK folder to your xcode project"](https://github.com/ivpusic/react-native-image-crop-picker#only-if-you-are-not-using-cocoapods) -- but to which folder in the Xcode project?) and [extra things need to be done before releasing a production build](https://github.com/ivpusic/react-native-image-crop-picker#manual). It's a shame as it looks like a lot of effort has been put into this package.
